tests/spec/fs.appendFile.spec.js

```diff
@@ -125,3 +125,168 @@ describe('fs.appendFile', function() {
     });
   });
 });
+
+
+describe('fs.promises.appendFile', function() {
+  beforeEach(function(done) {
+    util.setup(function() {
+      var fs = util.fs();
+      fs.promises.writeFile('/myfile', 'This is a file.', { encoding: 'utf8' })
+        .then(done())
+        .catch(function (error)
+        {
+          throw error;
+        }); 
+    });
+  });
+  afterEach(util.cleanup);
+
+  it('should be a function', function() {
+    var fs = util.fs();
+    expect(fs.promises.appendFile).to.be.a('function');
+  });
+
+  it('should append a utf8 file without specifying utf8 in appendFile', function(done) {
+    var fs = util.fs();
+    var contents = 'This is a file.';
+    var more = ' Appended.';
+
+    fs.promises.appendFile('/myfile', more)
```

Comment:
Add a `return` to the front of this, so you return this promise (instead of needing/using `done`).